billy-portugal/src/main/java/com/premiumminds/billy/portugal/services/export/PTCreditNoteData.java

```diff
@@ -43,10 +43,11 @@ public PTCreditNoteData(
         final BigDecimal amountWithoutTax,
         final String settlementDescription,
         final String hash,
-        final String qrCodeString) {
+        final String qrCodeString,
+		final String atcud) {
```

Comment:
Your changes across the files are using a different indentation format than the existing code (tabs vs spaces). Can you review them to follow the same standard as currently in use on the billy project?